vcpkg.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": [
     {
       "name": "blosc",
-      "version>=": "1.21.5"
+      "version>=": "1.21.6"
```

Comment:
is there a reason you're upgrading the blosc version requirement as part of a README change?

Reply:
Matching my environment with the documentation and the requirements. The previous release was over a year ago.